changelogs/fragments/1453-add-support-for-keyed_groups-to-linode-inventory-plugin.yml

```
@@ -0,0 +1,4 @@
+minor_changes:
+    - linode inventory plugin - add support for ``keyed_groups``, ``groups``,
+      and ``compose`` options 
+      (https://github.com/ansible-collections/community.general/issues/1326)
```

```suggestion
      (https://github.com/ansible-collections/community.general/issues/1326).
```
Sorry, I apparently overlooked that.